mod/wikirate/lib/javascript/script_wikirate_coffee.js.coffee

```diff
@@ -124,6 +124,23 @@ $(window).ready ->
   $('body').on 'change', '.SELF-company_comparison select', ->
     $(this).closest('form').submit()
 
+  # $("body").on "click", ".TYPE-project , ->
+  #   alert "add item link"
+  #   anchor = $(this)
+  #   parent = anchor.closest(".TYPE-project").
+  #   if parent
+
+decko.slotReady (slot) ->
+  return unless slot.hasClass("TYPE-project") && slot.find("form")
+  parent = slot.find(".RIGHT-parent .pointer-item-text")
+  appendParentToAddItem parent
+
+appendParentToAddItem = (parent) ->
+  return unless parent.val()
+  anchor = parent.slot().find "._add-item-link"
+  new_href =  anchor.attr("href") + "&" + $.param({ "filter[project]" : parent.val() })
+  anchor.attr "href", new_href
```

Comment:
Not sure I like this solution.  We're going to need filter presets in many of our use cases, and this is a vary narrow way to achieve that.